### ISSUE TITLE
Error when resetting password before registration

### ISSUE THREAD

**Issue by Kodzhabashev** [humdek-unibe-ch/sh-selfhelp#220]
In GitLab by @olivierbur on Dec 19, 2019, 17:43

A student was a little bit impatient and tried to reset the password before getting the link of the registration for the selfhelp platform (Hermes). After he got the link, neither worked, not the link or resetting the password.

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Jan 14, 2020, 09:43

Sorry for the late reply. A few questions:

1. Did you create the user manually in the CMS?
2. Did the user register with a code that you provided?
3. What was the error message when attempting to reset the password the first time?
4. What was the error message after the link was clicked?
5. What was the error message when attempting to reset the password after the link was clicked?
6. What is the email address of the user or the code assigned to the user?

**Comment by Kodzhabashev** [comment]
In GitLab by @olivierbur on Jan 14, 2020, 09:52

No worries!

1. No, we piloted the registration. Participants received the code via e-mail and had to register on the hermes platform.
2. yes.
3. I don't know
4. I don't know
5. I don't know
6. chen.xie@students.unibe.ch

I just realised however, that maybe this user already existed, when he tried to log in. I will contact the user and let you know, so you don't try to fix this for nothing.
